Packs/Threatgraph/Integrations/Threatgraph/README.md

```diff
@@ -0,0 +1,61 @@
+# V2.1.2 no change, just reversioned following git restructuring and changed tags for deployment
+V1.1.1 support multiple sensors
+V1.1.0 added test method
+V1.0.0 original version
+A script to get the parent and child process data from crowdstrike.
+This integration was integrated and tested with version xx of Threatgraph
```

Comment:
Please insert xx version or remove this line.

Reply:
description modified/updated in last pull. Note, we actually use the description field to show our changelog because XSOAR provides no other means of doing this in the front end.